sys/kern/sys_process.c

```diff
@@ -492,6 +492,14 @@ proc_readmem(struct thread *td, struct proc *p, vm_offset_t va, void *buf,
 	return (proc_iop(td, p, va, buf, len, UIO_READ));
 }
 
+ssize_t
+proc_readmem_cap(struct thread *td, struct proc *p, vm_offset_t va, void *buf,
+    size_t len)
+{
+
```

Comment:
delete this line, no need to propogate obsolete style even though the neighboring functions use it